driver/src/main/java/org/neo4j/driver/internal/util/CertificateTool.java

```diff
@@ -147,6 +163,43 @@ public static void loadX509Cert( Certificate cert, String certAlias, KeyStore ke
     {
         keyStore.setCertificateEntry( certAlias, cert );
     }
+
+    private static X509Certificate generateSelfSignedCertificate(X500Name name, int bits, long validSecs)
```

Comment:
It was a java7/java8 issue - the original class used here had been moved in Java 8, so it can't be compiled on Java 8. +1 on not using the internal classes though, this was just a quick fix. 

However, since this is just for testing, maybe we can follow up with a commit that moves this tool to tests, and we can pop in a test dependency that can generate these certificates?